sites-enabled/30-chinese.freecodecamp.dev.conf

```diff
@@ -59,3 +66,27 @@ server {
   # additional config
   include snippets/common/general.conf;
 }
+
+# non-www, subdomains redirect
+server {
+  listen 443 ssl http2;
+  listen [::]:443 ssl http2;
+
+  server_name .freecodecamp.dev;
+
+  # SSL
+  include snippets/common/ssl-freecodecamp-org.conf;
+
+  return 302 https://chinese.freecodecamp.dev$request_uri;
+}
+
+# HTTP redirect
+server {
+  listen 80;
+  listen [::]:80;
+
+  server_name .freecodecamp.dev;
+
+  return 302 https://chinese.freecodecamp.dev$request_uri;
+}
```

Comment:
```suggestion

# The HTTP Block is commented for this domain
# Listening on Port 80 is not possible for .dev domains because of HSTS Policy

# HTTP redirect
# server {
#   listen 80;
#   listen [::]:80;

#   server_name .freecodecamp.dev;

#   return 302 https://chinese.freecodecamp.dev$request_uri;
# }
```

Reply:
Comment out port 80 changes.